src/detect-reference.c

```diff
@@ -162,6 +165,8 @@ static DetectReference *DetectReferenceParse(const char *rawstr, DetectEngineCtx
         SCLogError("strdup failed: %s", strerror(errno));
         goto error;
     }
+    /* already bound checked to be REFERENCE_SYSTEM_NAME_MAX or less */
```

Comment:
Are we not mentioning the wrong const here? Shouldn't it be `REFERENCE_CONTENT_NAME_MAX` ?

Reply:
good catch. I thought I had addressed this. Will make an update.